src/user-guide/installing-workflows.rst

```diff
@@ -0,0 +1,370 @@
+.. _Installing-workflows:
+
+Installing Workflows
+====================
+
+Cylc commands identify workflows via their names, which are relative path names
+under the :term:`cylc-run directory`, ``~/cylc-run/`` by default.
+
+Workflows can be grouped together under sub-directories. E.g.:
+
+.. code-block:: console
+
+   $ cylc scan --state=all --name nwp --format=tree
+   nwp
+   ├─oper
+   │ ├─ region1
+   │ └─ region2
+   └─test
+      └─ region1
+
+This chapter will demonstrate how to install a workflow from an arbitrary
+location, called a :term:`source directory`.
+``cylc install`` will create a new directory in the :term:`cylc-run directory`
+for each installation of a workflow.
+
+.. _Install-Workflow:
+
+The Cylc Install Command
+------------------------
+
+Workflow names can be installed with the ``cylc install`` command,
+which creates the :term:`run directory` structure and some service
+files underneath it. Otherwise, ``cylc play`` will do this at workflow start
+up.
+
+Once you have written your workflow, you can have Cylc install the workflow for
+you, using the ``cylc install`` command.
+
```

Comment:
```suggestion

For example, if you had written a workflow in ``/mydrive/workflows/example1``
you would run

.. code:: bash
   $ cd /mydrive/workflows/example1   # Command line options can allow you to skip being in the source directory.
   $ cylc install


```

Reply:
I originally had an example in like this but, given the various options for cylc install, and depending on the pwd, I thought the following few lines (and example table) explained it better to users. 
How do you feel about me instead renaming the section below, from `Command Line Options` to `Using Cylc Install` instead? Just thinking of the over-arching flow of the read of the article.

Reply:
I thought I'd deleted this comment, I decided it was unecessary after reading the next bit: I think your suggestion is a good idea: "using cylc install"